playback/studio/studio.py

```diff
@@ -55,8 +79,14 @@ def _play_category(self, category):
         except Exception as ex:
             _logger.info(u'Cannot tune equalizer for category {} - {}'.format(category, ex))
             return ex
-        equalizer = Equalizer(find_matching_playable_recordings(
-            self.tape_recorder, tuning.playback_function, category,
-            self.lookup_properties), tuning.result_extractor, tuning.comparator,
-            comparison_data_extractor=tuning.comparison_data_extractor)
+
+        if recording_ids:
+            playable_recordings = by_id_playable_recordings(
```

Comment:
Function names should be verbs/actions

Reply:
this is not part of this PR and it is already used by users of the library so it is a breaking change